### PR TITLE
README.md Updated typo fixes done.

### PR DESCRIPTION
**Pull request by prasadjivane**
Updated README.md typo fixes.

#### Mandatory Checklist
**PRs will only be reviewed after checklist is complete**

- [ ] Add input/output [type hints](https://docs.python.org/3/library/typing.html) to new functions/methods
- [ ] Write design document if needed ([Do I need to write a design document?](https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#design-document))
- [ ] Write/update unit tests
- [ ] Write/update integration tests
- [ ] Write/update functional tests if needed
- [ ] `make pr` passes
- [ ] `make update-reproducible-reqs` if dependencies were changed
- [ ] Write documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,9 @@
...
 
 The AWS Serverless Application Model (SAM) CLI is an open-source CLI tool that helps you develop serverless applications containing [Lambda functions](https://aws.amazon.com/lambda/), [Step Functions](https://aws.amazon.com/step-functions/), [API Gateway](https://aws.amazon.com/api-gateway/), [EventBridge](https://aws.amazon.com/eventbridge/), [SQS](https://aws.amazon.com/sqs/), [SNS](https://aws.amazon.com/sns/) and more. Some of the features it provides are:
-- **Initialize serverless applications** in minutes with AWS provided infrastructure templates with `sam init`
-- **Compile, build, and package** Lambda functions with provided runtimes and with custom Makefile workflows, for zip and image types of Lambda functions with `sam build`
-- **Locally test** a Lambda function and API Gateway easily in a Docker container with `sam local` commands on SAM and CDK applications
+- **Initialize serverless applications** in minutes with AWS-provided infrastructure templates with `sam init`
+- **Compile, build, and package** Lambda functions with provided runtimes and with custom Makefile workflows, for zipping and image types of Lambda functions with `sam build`
```
Comment (hawflau) [inline README.md:15]: not sure why you change it to a gerund. "zip" is a noun in this sentence (as in "zip type Lambda Function").

File: README.md [hunk 2]
```diff
@@ -11,9 +11,9 @@
...
+- **Locally test** a Lambda function and API Gateway quickly in a Docker container with `sam local` commands on SAM and CDK applications
```
Comment (hawflau) [inline README.md:16]: "easily" makes sense to me. And this is not a typo. Why is changing to "quickly" more appropriate?

File: README.md [hunk 3]
```diff
@@ -33,16 +33,16 @@ environment that lets you locally build, test, debug, and deploy [AWS serverless
 
 
 **Next Steps:** Learn to build a more complex serverless application.
-* [Extract text from images and store in a database](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-example-s3.html) using Amazon S3 and Amazon Rekognition services.
-* [Detect when records are added to a database](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-example-ddb.html) using Amazon DynamoDB database and asynchronous stream processing.
+* [Extract text from images and store it in a database](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-example-s3.html) using Amazon S3 and Amazon Rekognition services.
+* [Detect when records added to a database](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-example-ddb.html) using Amazon DynamoDB database and asynchronous stream processing.
```
Comment (hawflau) [inline README.md:37]: I believe removing "are" is not grammatically correct.

File: README.md [hunk 4]
```diff
@@ -33,16 +33,16 @@ environment that lets you locally build, test, debug, and deploy [AWS serverless
...
 * [Explore popular patterns](https://serverlessland.com/patterns)
 
 
 ## What is this Github repository? 💻
-This Github Repository contains source code for SAM CLI. Here is the development team talking about this code:
+This Github repository contains source code for SAM CLI. Here is the development team talking about this code:
 
 > SAM CLI code is written in Python. Source code is well documented, very modular, with 95% unit test coverage. 
-It uses this awesome Python library called Click to manage the command line interaction and uses Docker to run Lambda functions locally.
+It uses this excellent Python library called Click to manage the command line interaction and uses Docker to run Lambda functions locally.
```
Comment (hawflau) [inline README.md:45]: this doesn't look like a typo to me. Why change it?

File: README.md [hunk 5]
```diff
@@ -70,17 +70,16 @@ to get started.
...
 
-Add new commands or enhance existing ones, report bugs, or request new features for the SAM CLI.
+Add new commands, enhance existing ones, report bugs, or request new features for the SAM CLI.
 Source code is located on Github at [awslabs/aws-sam-cli](https://github.com/awslabs/aws-sam-cli). Read the [SAM CLI Contributing Guide](https://github.com/awslabs/aws-sam-cli/blob/develop/CONTRIBUTING.md) to 
 get started. 
 
 **Update SAM Developer Guide**
 
-[SAM Developer Guide](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/index.html) provides comprehensive getting started guide and reference documentation.
+[SAM Developer Guide](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/index.html) provides a comprehensive getting started guide and reference documentation.
```
Comment (hawflau) [inline README.md:79]:
```suggestion
[SAM Developer Guide](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/index.html) provides a comprehensive getting started guide and a reference documentation.
```

**Review by hawflau — CHANGES_REQUESTED** [review]
Thanks for your contribution. I left some questions in which I don't think those changes are necessary. Could your please review and update accordingly?

**Review by hawflau — APPROVED** [review]

**Review by mndeveci — APPROVED** [review]

**Pull request merged**